src/amsterdam_schema/publish/cli.py

```diff
@@ -148,10 +150,98 @@ def replace_schema_base_url(temp_dir: Path, schema_base_url: str) -> None:
             if file_path.suffix == ".json":
                 with in_place.InPlace(root_path / file_name) as file:
                     for line in file:
-                        line = line.replace(DEFAULT_BASE_URL, schema_base_url)
+                        line = line.replace(DEFAULT_BASE_URL, schema_base_url)  # type: ignore
                         file.write(line)
 
 
+def azure_blob_uploader(
+    files_root: Path,
+    schema_pub_paths: List[List[str]],
+    doc_pub_paths: List[List[str]],
+    container: str,
+    index_file_obj: BytesIO,
+) -> None:
+
+    json_content_settings = ContentSettings(content_type="application/json")
+    html_content_settings = ContentSettings(content_type="text/html")
+    credential = os.getenv("SCHEMAS_SA_KEY")
+
+    blob_srv = BlobServiceClient(
+        f"https://{SCHEMAS_SA_NAME}.blob.core.windows.net", credential=credential
+    )
+
+    # First delete all blobs in the container
+    container_client = blob_srv.get_container_client("schemas")
+
+    container_client.delete_blobs(
+        *[b.name for b in container_client.list_blobs()]
```

Comment:
No need to materialize the list:

```suggestion
        *(b.name for b in container_client.list_blobs())
```